app/renderer/components/tabs/tabPage.js

```diff
@@ -32,23 +32,6 @@ class TabPage extends React.Component {
     this.onMouseLeave = this.onMouseLeave.bind(this)
```

Comment:
yes, you can try `npm run test -- --grep="tab pages tab page previews hovering over a tab page changes it"`